packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.styles.ts

```diff
@@ -5,26 +5,27 @@ import { getRTL } from '../../Utilities';
 const GlobalClassNames = {
   root: 'ms-Shimmer-container',
   shimmerWrapper: 'ms-Shimmer-shimmerWrapper',
+  shimmerGradient: 'ms-Shimmer-shimmerGradient',
   dataWrapper: 'ms-Shimmer-dataWrapper'
 };
 
-const BACKGROUND_OFF_SCREEN_POSITION = '1000%';
+const BACKGROUND_OFF_SCREEN_POSITION = '100%';
```

Comment:
Why this change needed?

Reply:
The 1000% version works different for translates. The 100% moves from offscreen on the left to offscreen on the right in 1.5 seconds. It should look the same as the current version. Is there a design reference I can look at for the specific timing and animation curve?